<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="NewLabel" name="labelFox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>691</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;The quick brown fox likes CSE 335.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelButtons">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Family</string>
    </property>
   </widget>
   <widget class="NewSlider" name="sliderSize">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAriel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ariel</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTNR">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>230</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Times New Roman</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonComic">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Comic Sans MS</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFont">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>small</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>big</string>
     </property>
    </item>
   </widget>
   <widget class="NewLineEdit" name="lineEditFont">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NewLabel</class>
   <extends>QLabel</extends>
   <header location="global">newlabel.h</header>
  </customwidget>
  <customwidget>
   <class>NewLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">newlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>NewSlider</class>
   <extends>QSlider</extends>
   <header location="global">newslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
